interface/interface.dm

```diff
@@ -1,12 +1,14 @@
 //Please use mob or src (not usr) in these procs. This way they can be called in the same fashion as procs.
-/client/verb/wiki()
+/client/verb/wiki(query as text)
 	set name = "wiki"
-	set desc = "Visit the wiki."
+	set desc = "Type what you want to know about.  This will open the wiki on your web browser."
 	set hidden = 1
 	if(config.wikiurl)
-		if(alert("This will open the wiki in your browser. Are you sure?",,"Yes","No")=="No")

```

Comment:
If you click the WIKI button, it's guaranteed to load a web page instead of asking if you want to first. Note how there's no replacement for this "Are you sure?" prompt.